doc/src/appendices/suiterc-config-ref.rst

```diff
@@ -2024,6 +2024,15 @@ earlier in the task job script, and variable assignment expressions can
 use cylc utility commands because access to cylc is also configured
 earlier in the script.  See also :ref:`TaskExecutionEnvironment`.
 
+You can also specify task execution parameter environment templates here (only
+relevant for *parameterized tasks* - see :ref:`Parameterized Tasks Label`.
```

Comment:
```suggestion
You can also specify job environment templates here for *parameterized tasks* (see :ref:`Parameterized Tasks Label`).
```
"Task execution parameter environment template" is a bit of a mouthful and not easy to understand (not sure why we used to use "task execution environment'' instead of "job environment"...